src/Surfnet/StepupSelfService/SelfServiceBundle/Resources/views/Registration/partial/secondFactor.html.twig

```diff
@@ -13,7 +13,16 @@
         </p>
         <hr>
         <p>
-            {{ ('ss.registration.selector.'~type~'.description')|trans }}
+            {{
+                ('ss.registration.selector.'~type~'.description')|trans(
+                    {
+                        '%tiqr_android_link_start%': '<a href="'~tiqrAppAndroidUrl~'">',
```

Comment:
Good, I thought I was going mad. Well in that case, if you want to do the "correct" thing, there's also [`|replace`](http://twig.sensiolabs.org/doc/filters/replace.html) :stuck_out_tongue: (in other words: `|trans|e|replace(...)|raw`)